### PR TITLE
Docs: Fix the example to actually render a cube

### PR DESCRIPTION
**Pull request by oskar1233**
**Description**

The example wasn't rendering the promised cube, this PR fixes it.

Live example in JSFiddle works fine so I took the BoxGeometry arguments from there.

### CODE REVIEW

**Comment by mrdoob** [comment]
Thanks!

File: docs/manual/en/introduction/Creating-a-scene.html [hunk 1]
```diff
@@ -70,7 +70,7 @@ <h2>Creating the scene</h2>
 		<p><em>"That's all good, but where's that cube you promised?"</em> Let's add it now.</p>
 
 		<code>
-		const geometry = new THREE.BoxGeometry();
+		const geometry = new THREE.BoxGeometry(1, 1, 1);
```
[the remark this answers is not in the document]
Comment (oskar1233) [reply docs/manual/en/introduction/Creating-a-scene.html]: :grimacing: thanks

**Comment by oskar1233** [comment]
@Mugen87 I installed `three` from npm so I'm probably not on `dev` branch - should that then be a hotfix and merged to master?

**Comment by Mugen87** [comment]
TBH, I think this PR should be reverted.

**Comment by Mugen87** [comment]
Okay, but then I'll update the localized pages, too.

**Pull request merged**